src/middlewared/middlewared/etc_files/generate_ssl_certs.py

```diff
@@ -19,9 +20,23 @@ def write_certificates(certs):
             with open(cert['csr_path'], 'w') as f:
                 f.write(cert['CSR'])
 
+    """
+    Write unified CA certificate file for use with LDAP.
+    """
+    shutil.copyfile('/usr/local/share/certs/ca-root-nss.crt',
```

Comment:
What are all these extra certs needed for? Shouldn't the uploaded certificates be enough to trust the user's CA?

Reply:
Perhaps only do this when the user has not provided their own CA certificates. I just don't like the idea of being forced to trust someone else's CA when I have intentionally created my own root of trust.

Reply:
Done.